Week-2/Assignment2.2/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
```

Comment:
CSS can be improved please refer 
https://github.com/pesto-students/p9-rahul-RahulSG39/pull/4/files